lm_eval/tasks/custom_metrics/multiple_metrics/libeval.py

```diff
@@ -0,0 +1,43 @@
+import os
```

Comment:
We have been meaning to stop using this file. The `safe_subprocess` module is a lot more robust.  A few of the evals still use this.